ios/FluentUI/Button/FluentButtonToggleStyle.swift

```diff
@@ -33,17 +33,38 @@ public struct FluentButtonToggleStyle: ToggleStyle {
     }
 
     private var buttonTokens: [ButtonToken: ControlTokenValue] {
-        [
+        var tokens: [ButtonToken: ControlTokenValue] = [
             .cornerRadius: .float { GlobalTokens.corner(.radius40) }
         ]
+
+        if let tokenOverrides = tokenOverrides {
+            tokens = tokens.merging(tokenOverrides) { (_, new) in new}
+        }
+
+        return tokens
     }
 
     private var buttonOnTokens: [ButtonToken: ControlTokenValue] {
         let backgroundColor = fluentTheme.color(.brandBackgroundTint)
-        return buttonTokens.merging([
+        var tokens: [ButtonToken: ControlTokenValue] = buttonTokens.merging([
             .backgroundColor: .uiColor { backgroundColor },
             .backgroundPressedColor: .uiColor { backgroundColor },
             .backgroundFocusedColor: .uiColor { backgroundColor }
         ]) { (_, new) in new }
+
+        if let tokenOverrides = tokenOverrides {
+            tokens = tokens.merging(tokenOverrides) { (_, new) in new }
+        }
+
+        return tokens
+    }
+
+    private var tokenOverrides: [ButtonToken: ControlTokenValue]?
+}
+
+public extension FluentButtonToggleStyle {
+    /// Provide override values for various `ButtonToken` values.
+    mutating func overrideTokens(_ overrides: [ButtonToken: ControlTokenValue]) {
```

Comment:
Now that I'm thinking about it, any chance we could get a quick test in the demo controller for this function? Similar to how we already test the equivalent on FluentButtonStyle

Reply:
Added a demo which showcases the overrides for both the button and toggle styles. Added a demo to the description.